tests/complex_fluids/cf_forcing_op_01_2d.cell.logarithm.output

```diff
@@ -1,6 +1,6 @@
 AdvDiffHierarchyIntegrator::initializePatchHierarchy(): tag_buffer = 0
 Conformation tensor is SPD
 Error norms:
-  L1-norm:  0.077081549
-  L2-norm:  0.049351098
-  max-norm: 0.040101043
+  L1-norm:  0.97983536
+  L2-norm:  1.0673658
+  max-norm: 2.7850786
```

Comment:
Should these error norms be so high?

Reply:
Yes. These are absolute norms. The relative norms are ~0.05 plus this is a fairly coarse grid.